packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js

```diff
@@ -0,0 +1,80 @@
+import { LABELS, TRACKING_BASE } from './constants';
+
+export default class OvhManagerNetAppNetworkConfigurationCtrl {
+  /* @ngInject */
+  constructor($translate, Alerter, NetappNetworkConfigurationService) {
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.NetappNetworkConfigurationService = NetappNetworkConfigurationService;
+    this.LABELS = LABELS;
+    this.TRACKING_BASE = TRACKING_BASE;
+  }
+
+  $onInit() {
+    this.disableVrackServicesField = false;
+    this.disableSubnetField = true;
+
+    if (!this.vrackServices?.length) {
+      const noVrackServices = {
+        display: {
+          nameWithVrackId: this.$translate.instant(
+            'netapp_network_configuration_no_vrack_service_field',
+          ),
+        },
+      };
+      this.vrackServices.push(noVrackServices);
+      this.selectedVrackService = noVrackServices;
+      this.disableVrackServicesField = true;
+    }
+  }
+
+  onVrackServiceSelected(vrackService) {
+    this.selectedSubnet = null;
+    this.subnets = JSON.parse(
+      JSON.stringify(vrackService.currentState.subnets),
+    );
+
+    if (!this.subnets.length) {
```

Comment:
Can't be null here ?
```suggestion
    if (!this.subnets?.length) {
```